### DISCUSSION TITLE
Use lightbox with videos

### DISCUSSION THREAD

**Discussion by wilcar** [quarto-dev/quarto-cli#9290] [Feature Requests]
### Description

Quarto uses the [GLightbox javascript library](https://biati-digital.github.io/glightbox/) to add lightbox styling and behavior to images in HTML documents .
The GLightbox javascript library can deal with videos.

What I tried according to the [documentation](https://github.com/biati-digital/glightbox)  : 

    ---
    title: "test lightbox video"
    editor: visual
    ---
    ```{=html}
    <a href="https://vimeo.com/115041822" class="glightbox"> <img src="xxx.png" alt="image"/><img    src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/1b/R_logo.svg/310px-R_logo.svg.png"/>
    </a>
    ```

When I click on the "R" image, the video is open in full screen but I dont get the close button and not the same appeareance that the images lightbox.

**Comment by cderv** [comment]
I don't think we support video in our Lightbox  integration (https://quarto.org/docs/output-formats/html-lightbox-figures.html) 

When running your example, I also don't think that the lightbox JS scripts are loaded 

So for now integrated Lightbox in quarto is only for Images.
